LWCE_Core/Src/XComLongWarCommunityEdition/Classes/LWCETacticalVisibilityHelper.uc

```diff
@@ -191,6 +193,10 @@ function UpdateVisibilityMarkers()
         HideAllVisibilityMarkers();
         return;
     }
+    if (Cursor().Location != m_vLastValidCursor || vDestination != m_vLastValidDestination)
```

Comment:
I'm going to merge this as-is because it would be silly to request a new version for this, but a very tiny style nitpick for future reference - ending braces in LWCE should always be followed by a blank line. (I should really write up an actual style guide at some point.)